### ISSUE TITLE
Dots appearing in first column of indentation

### ISSUE THREAD

**Issue by danderson00** [Microsoft/vscode#48098]
Issue Type: <b>Bug</b>

Please see screenshot. I'm unable to identify what setting affects this.

VS Code version: Code 1.22.1 (950b8b0d37a9b7061b6f0d291837ccc4015f5ecd, 2018-04-06T00:21:13.607Z)
OS version: Darwin x64 17.4.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz (8 x 2200)|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (3.09GB free)|
|Process Argv|/Applications/Visual Studio Code.app/Contents/MacOS/Electron|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
prettier-vscode|esb|1.3.0

</details>

![screen shot 2018-04-18 at 3 08 24 pm](https://user-images.githubusercontent.com/1265935/38912760-7b69e9f8-431a-11e8-9909-c3c110f09cfc.png)

**Comment by shizengzhou** [comment]
`"javascript.suggestionActions.enabled": false` from https://code.visualstudio.com/updates/v1_22#_suggestion-code-actions-for-javascript-and-typescript

**Comment by ramya-rao-a** [comment]
- Do you still see the same when you run VS Code with extensions disabled? Try running `code --disable-extensions`.
- Do you see the same in the latest [Insiders](https://code.visualstudio.com/insiders)?
- Do you see this in all file types?

**Comment by danderson00** [comment]
`"javascript.suggestionActions.enabled": false` fixed this. It would seem that what I was seeing was a suggestion action on every line, though no suggestion was appearing. Now that I think about it I do remember seeing another issue where suggestions were appearing for every line.

It only occurred for JS files - the suggestion that appeared after disabling extensions was to convert CommonJS to ES6 modules.

Strange thing is, after running VS Code with the `--disable-extensions` flag, it doesn't occur, even after restarting with extensions enabled. My settings are as follows:

```
{
    "workbench.startupEditor": "newUntitledFile",
    "editor.minimap.enabled": false,
    "explorer.openEditors.visible": 0,
    "editor.autoIndent": false,
    "editor.tabSize": 2,
    "explorer.confirmDragAndDrop": false,
    "extensions.ignoreRecommendations": false,
    "prettier.semi": false,
    "prettier.singleQuote": true,
    "prettier.printWidth": 120,
    "editor.formatOnSave": false,
    "telemetry.enableTelemetry": false,
    "editor.autoClosingBrackets": false,
    "editor.folding": false,
    "editor.quickSuggestions": {
        "other": false,
        "comments": false,
        "strings": false
    },
    "editor.parameterHints": false,
    "editor.lightbulb.enabled": false,
    "editor.suggestOnTriggerCharacters": false,
    "javascript.suggestionActions.enabled": false
}
```

**Comment by mjbvz** [comment]
Duplicate of #47346. Should be fixed in 1.22.2 recovery build
